frontend/src/routes/explore/+page.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+    import data from "$lib/data";
+</script>
+
+<div class="columns">
```

Comment:
`class="columns my-0" style="height: calc(100vh-56px);"` might help :)

Reply:
Got it. Nice one :D